src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './calculator.css';
+
+class Pad extends React.Component {
+  render {
```

Comment:
- [ ] Your app is not running because there is an issue with the code, your code is missing () after the render. Please fix it.
    - You can consider using this code `render () {`

### See the image of error 

![image](https://user-images.githubusercontent.com/83803414/164450930-959bb74d-2451-43ce-b3dc-1ee0998124f1.png)